slp_tf/tests/integration/test_tf_processor.py

```diff
@@ -317,20 +300,22 @@ def test_resources_with_same_name(self):
         assert len(otm.components) == 2
         assert otm.components[0].name == otm.components[1].name
 
-    def test_backward_compatibility(self):
+    @pytest.mark.parametrize('mapping_file', [
+        pytest.param(get_data(terraform_iriusrisk_tf_aws_mapping), id="with actual mapping file"),
+        pytest.param(get_data(terraform_iriusrisk_tf_aws_mapping_v180), id="with backwards mapping_file")])
+    def test_backward_compatibility(self, mapping_file):
```

Comment:
Should we change the name of this test? now it iuses the newest mapping file:
maybe test_aws_security_groups_components_full_example

Reply:
The test's name has been changed to the suggested one.